app/src/main/kotlin/com/adesso/movee/internal/util/api/NetworkCallAdapter.kt

```diff
@@ -97,6 +97,7 @@ private class ResultCall<R> constructor(
             override fun onFailure(call: Call<R>, throwable: Throwable) {
                 Timber.e(throwable)
                 val error = when (throwable) {
+                    is Failure -> throwable
```

Comment:
isn't this too generic?

Reply:
if Failure object is already created beforehand, there is no need to map it to other type.